Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js

```diff
@@ -161,9 +161,11 @@ export default function Homepage() {
       <StudentAccess/>
       <EthisimIntro/>
 
+    {/*}
       <Box className={classes.copyright}>
```

Comment:
Why?

Reply:
I commented it out back when we weren't sure about using it; I'll delete it on my branch for now